docs/styles.rst

```diff
@@ -0,0 +1,15 @@
+Styles
+======
+
+VSG supports several predefined styles.
+They can be accessed by using the **--style** command line option.
```

Comment:
How about 'enabled'?

Reply:
Might be confusing as we use enable/disable terms for rules, and styles are not rules.

Reply:
utilized?

Reply:
We have come to the point where I do not know. I am not a native speaker, and I am not sure which one would be better `used` or `utilized`.

Reply:
lol...I am a native speaker and it isn't clear to me either.  I think I will go with used.  I updated the entire `style` documentation.  Hopefully it will be clearer.